ydb/core/tx/schemeshard/schemeshard__table_stats.cpp

```diff
@@ -5,6 +5,23 @@
 #include <ydb/core/base/cputime.h>
 #include <ydb/core/protos/sys_view.pb.h>
 
+namespace {
+
+TVector<TString> MapChannelsToStoragePoolKinds(const NKikimr::TStoragePools& pools, const NKikimr::TChannelsBindings& bindings) {
```

Comment:
@ijon Is it ok to calculate this mapping each time we receive TEvPeriodicTableStats? (I think yes, because there is usually only a handful of storage pools in a database and channels in a datashard.) The other option I see is to save it in TSubDomainInfo in SchemeShard.

Reply:
You can't save it in TSubDomainInfo, since TChannelsBinding is different for each tablet.